src/import-export/aqb/gnc-ab-utils.c

```diff
@@ -384,6 +384,40 @@ gnc_ab_get_purpose(const AB_TRANSACTION *ab_trans)
 gchar *
 gnc_ab_description_to_gnc(const AB_TRANSACTION *ab_trans)
 {
+    gchar *result;
+    gchar *transactionText = gnc_ab_transactionText_to_gnc(ab_trans);
+    gchar *purpose = gnc_ab_purpose_to_gnc(ab_trans);
+
+    if (transactionText)
+    {
+        result = g_strjoin("; ", transactionText, purpose, (gchar*) NULL);
+        g_free(transactionText); transactionText = NULL;
```

Comment:
Ah, yeah, sorry, that's how I do this usually as I consider this as "one transaction". Hence it escaped my attention. But of course you're right. I'll fix that.